.gitignore

```diff
@@ -11,4 +11,5 @@ atlassian-ide-plugin.xml
 *.iml
 
 Google Docs Share/src/main/amp/config/alfresco/web-extension/custom-slingshot-googledocs-context.xml
-Google Docs Repository/src/main/oauth-client-resources/**
\ No newline at end of file
+Google Docs Repository/src/main/oauth-client-resources/**
+Google Docs Repository/src/main/amp/config/alfresco/subsystems/googledocs/drive/google-customResponse-context.xml
```

Comment:
I notice that we also have:

https://github.com/Alfresco/googledrive/tree/develop/Google%20Docs%20Repository/src/main/customResponseContextFiles

Related ?

Reply:
Sort of related, the one that I added to gitignore is [copied by a maven plugin in another location at build time](https://github.com/Alfresco/googledrive/blob/e2304b9ae054ed98cbc2e2c4f5527ff2f58237ef/Google%20Docs%20Share/pom.xml#L151).